ckanext/geocat/tests/fixtures/geocat-testdata.xml

```diff
@@ -11990,7 +11981,7 @@
                                     <gco:CharacterString>Map Preview</gco:CharacterString>
                                     <gmd:PT_FreeText>
                                         <gmd:textGroup>
-                                            <gmd:LocalisedCharacterString locale="#DE">Map Preview
+                                            <gmd:LocalisedCharacterString locale="#DE">Kartenvorschau
```

Comment:
why don't we change on the l.12002 also to `Kartenvorschau`?

Reply:
What do you mean by  `l.12002` ?

Reply:
Done